blue_ai/envs/custom_decay.py

```diff
@@ -0,0 +1,26 @@
+import torch.nn as nn
+import torch
+from torch.nn.modules import MSELoss, Sequential
+
+
+class PositivePenaltyLoss(nn.Module):
+    known = MSELoss()
+
+    """
+    Uses a combinations of MSE Loss with a RELU penalty, meaning that positive 
+    weights will always have a large penalty 
+    """
+
+    def __init__(self, alpha=0.2):
+        super(PositivePenaltyLoss, self).__init__()
+        self.policy_hook: Sequential = None
```

Comment:
This is because of the deep copies that occur here; it stops us from passing a reference down, and instead, we need to get the reference back up. I know how disgusting this is, but I am not sure how else to deal with it.  
https://github.com/echalmers/blue_ai/blob/38b2dc8ef6f284df6323f93cd9d3b73f9541df1e/blue_ai/agents/dqn.py#L115-L116